scripts/speakerdamage/channel_changer.lua

```diff
@@ -0,0 +1,262 @@
+-- channel changer
+--
+-- your own ghosts thru 
+-- late-night static and
+-- broken antenna frequencies
+--
+-- KEY 3: change channel  
+-- KEY 1 (HOLD): tv guide
+-- ENC 1: volume
+-- ENC 2: speed
+-- ENC 3: pitch
+--
+-- your display is not broken
+
+engine.name = 'Glut'
+local VOICES = 1
+voiceGate = 0
+shiftMode = 0
```

Comment:
can this be local or is it getting read elsewhere?

Reply:
I think local should be fine